news/_posts/2022-10-30-playtest-20221030.md

```diff
@@ -0,0 +1,91 @@
+    <source src="{{ '/images/news/20221031-td.ogv' | relative_url }}" type="video/ogg">
+    <img src="{{ '/images/news/20221031-td.png' | relative_url }}" width="700" />
+  </video>
+  <figcaption>Forces collide</figcaption>
+</figure>
+
+The fairly recent Dune film has brought in many players into Dune 2k, and a decently sized multiplayer community was born. With the feedback from the community, Dune 2k has received a number of balance and polish changes.
+
+<figure>
+  <video width="700" autoplay loop muted>
+    <source src="{{ '/images/news/20221031-dune.mp4' | relative_url }}" type="video/mp4">
+    <source src="{{ '/images/news/20221031-dune.ogv' | relative_url }}" type="video/ogg">
+    <img src="{{ '/images/news/20221031-dune.png' | relative_url }}" width="700" />
+  </video>
+  <figcaption>Storm is approaching!</figcaption>
+</figure>
+
+INTERMISSION: some cool stats
+ * 565 pull requests were merged since last release
+ * 533 issues were closed, bringing the total amount down by 136
+
+TS has also received its fair share of features and bug fixes. This time around the highlight is tilting on slopes.
+
+<figure>
+  <video width="700" autoplay loop muted>
+    <source src="{{ '/images/news/20221031-slope.mp4' | relative_url }}" type="video/mp4">
+    <source src="{{ '/images/news/20221031-slope.ogv' | relative_url }}" type="video/ogg">
+    <img src="{{ '/images/news/20221031-slope.png' | relative_url }}" width="700" />
+  </video>
+  <figcaption>OpenRA now supports tilting on slopes!</figcaption>
+</figure>
+
+<br>
+
+Some of the more visible new features and improvements include:
+ * New Red Alert missions
+ * An ingame encyclopedia in Dune 2000
+ * Measures against spamming in-game chat
+ * The ability to mute players
+ * Support for disabling the chat in replays
+
+Notable changes for mappers and modders include:
+ * Automatic map reloading in the background
+ * Changed the way speed is calculated for ground units
+ * New map format which includes the map preview in the map versioning (map hash)
+ * Loaders for more audio formats (e.g. mp3, ogg), and formats used by the remasters
+ * Many linting and documentation updates
+
+As well as some other changes worth mentioning:
+ * Toolchain and runtime changes to .NET 6
+ * A massive amount of upgrades to both ingame and client UI
+ * Many more hotkeys are now rebindable
+ * Substantial progress towards localisation
+
+This post has covered just a small subset of the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)), and we encourage anybody interested in finding out more to read the full list. Head on over to the [download page]({{ '/download' | relative_url }}) to install the new playtest, and let us know what you think in the comments below, on [our forum](https://forum.openra.net/), [community Discord server](https://discord.gg/UcUztfr), or [GitHub](https://github.com/OpenRA/OpenRA/issues).
```

Comment:
Next stable is https://github.com/OpenRA/OpenRA/wiki/Changelog not bleed.

Reply:
community Discord server is https://discord.openra.net

Reply:
> Next stable is https://github.com/OpenRA/OpenRA/wiki/Changelog not bleed.

Yes, I'll update it once we have that changelog

> community Discord server is https://discord.openra.net/

Good catch, I just copied it over from another page